lib/yaml_db/serialization_helper.rb

```diff
@@ -195,7 +196,19 @@ def self.table_record_count(table)
         ActiveRecord::Base.connection.select_one("SELECT COUNT(*) FROM #{Utils.quote_table(table)}").values.first.to_i
       end
 
-    end
+      # Just return the first column unless the second column is also an id, in
+      # which case it joins them for ordering
+      def self.table_id(table)
+        id = table_column_names(table).first
 
+        # If this looks like a standard HABTM join table, add the second id of
+        # a join table for ordering
+        if id =~ /_id$/ and table_column_names(table)[1] =~ /_id$/
+          id += ", #{table_column_names(table)[1]}"
```

Comment:
I think Arel accepts either but it might be a tad cleaner to return an array and skip the string interpolation.